pyanaconda/core/product.py

```diff
@@ -80,7 +80,19 @@ def get_product_values():
     # .buildstamp, environment, stupid last ditch hardcoded defaults.
     config = configparser.ConfigParser()
     config.add_section("Main")
-    config.set("Main", "IsFinal", os.environ.get("ANACONDA_ISFINAL", "false"))
+
+    # Get IsFinal property from /etc/os-release file
+    # FIXME https://issues.redhat.com/browse/INSTALLER-3587
+    with open('/etc/os-release') as file:
```

Comment:
You should / can now use the new API adam introduced: https://github.com/rhinstaller/anaconda/pull/6284

Reply:
ACK